statsmodels/discrete/tests/test_discrete.py

```diff
@@ -1772,6 +1773,244 @@ def test_predict_prob(self):
         assert_allclose(chi2[:], (0.64628806058715882, 0.98578597726324468),
                         rtol=0.01)
 
+class TestNegativeBinomial_pNB2Newton(CheckModelResults):
+    @classmethod
+    def setupClass(cls):
```

Comment:
master now uses pytest instead of nosetest.
this needs to be `setup_class` now, instead of camel case